lms/urls.py

```diff
@@ -995,6 +995,11 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    if getattr(settings, "EDX_GRADES_STORAGE_MEDIA_URL") and getattr(settings, "EDX_GRADES_STORAGE_MEDIA_ROOT"):
```

Comment:
If `EDX_GRADES_STORAGE_MEDIA_URL` param is missed is the `settings` `getattr` rises an AttributeError, either for `EDX_GRADES_STORAGE_MEDIA_ROOT` param

Reply:
Done